templates/taxi/driver_form.html

```diff
@@ -5,7 +5,7 @@
   <h1>{{ object|yesno:"Update,Create" }} driver license number</h1>
   <form action="" method="post" novalidate>
     {% csrf_token %}
-    {{ form|crispy }}
+    {{ form|crispy|linebreaksgit commit -am 'Solution of py-taxi-service-user-form-and-validation' }}
```

Comment:
commit -am 'Solution of py-taxi-service-user-form-and-validation' not here))

Reply:
Oops, sorry, this needs to be tracked...